test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,96 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Service Binding with Strimzi Kafka Cluster database without applicationSelector
```

Comment:
I believe this scenario is just collecting secret details from backend service so could it be renamed to something like Collect secrets from Strimzi Kafka Cluster backend service, or something else more appropriate?

Reply:
This is a great suggestion. Agreed, will make it more intuitive

Reply:
If this scenario does not provide application selector, then do we really need use Kafka as backing service or we could go with something lighter/less resource intensive? Also, the feature file here is about binding an apllication to service, but since we are missing the application, I would say that we should create a separate feature file.

Reply:
we have a scenario with lighter backend https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToService.feature#L200

Reply:
Kafka related scenario removed . A simpler Backend service acceptance test added instead